src/services/resources/logic.ts

```diff
@@ -27,6 +27,20 @@ export const getLoggedInLogic = async (userId: string) : Promise<ILogicResponse
     return response;
 };
 
+export const logoutLogic = async () : Promise<ILogicResponse | IErrorResponse> => {
+    const clearCookies: IClearCookie[] = [
+        {
+            name: "id_token"
+        },
+    ];
+    
+    const response: ILogicResponse = {
```

Comment:
should the response message here be defined? 

Right now, the front end just assumes that an empty string response means success

Reply:
nah this is fine for now